bemani/backend/ddr/base.py

```diff
@@ -127,6 +127,13 @@ def format_profile(self, userid: UserID, profile: Profile) -> Node:
         """
         return Node.void("game")
 
+    def format_profile_part(self, userid: UserID, profile: Profile, part: str) -> Node:
```

Comment:
Profile for SN2 is split into 3 binary chunks, where to break this logic down, it being here feels wrong

Reply:
Honestly the `format_profile` and `unformat_profile` convention were both established to just give some sort of unity across games. They don't need to exist outside of the fact that the base of most games assumes it can call that for profile loads. If there's 3 parts here, this can diverge for cleaner code.